src/api/handlers/guild.ts

```diff
@@ -587,6 +587,36 @@ export async function editEmbed(
   );
 }
 
+/** Returns the widget for the guild. */
+export async function getWidget(guildID: string, options?: { force: boolean }) {
+  const guild = await cacheHandlers.get("guilds", guildID);
+  if (!options?.force) {
+    if (!guild) throw new Error(Errors.GUILD_NOT_FOUND);
+    if (!guild?.widgetEnabled) throw new Error(Errors.GUILD_WIDGET_NOT_ENABLED);
+  }
+
+  return RequestManager.get(`${endpoints.GUILD_WIDGET(guildID)}.json`);
+}
+
+/** Returns the widget image URL for the guild. */
+export async function getWidgetImageUrl(
+  guildID: string,
+  options?: {
+    style?: "shield" | "banner1" | "banner2" | "banner3" | "banner4";
+    force?: boolean;
+  },
+) {
+  const guild = await cacheHandlers.get("guilds", guildID);
```

Comment:
This isn't used outside the if () so move it inside the if

Reply:
@itohatweb

Reply:
Ah completely overlooked that